summit/2018.html

```diff
@@ -55,7 +55,7 @@ <h2 class="support">Support Us</h2>
   <div class="row">
       <div class="col-md-10 col-md-offset-1">
         <br /><br />
-        <a href="#" class="coc">Code of Conduct</a>
+        <a href="https://opensourcedesign.net/code-of-conduct/" class="coc">Code of Conduct</a>
```

Comment:
Wondering if we should use relative links instead of absolute, but anyway, this would work! Thanks @AnXh3L0